src/cli/common.go

```diff
@@ -138,7 +138,10 @@ func Validate(projectState *state.State, logger *zap.SugaredLogger) error {
 
 func Rename(projectState *state.State, logger *zap.SugaredLogger, logEmpty, dryRun bool) error {
 	// Get plan
-	rename := plan.Rename(projectState)
+	rename, err := plan.Rename(projectState)
```

Comment:
Vid `src/manifest/manifest.go` ... 
`panic` som previedol na `error`, tak musim upravit aj vsetky metody, co na to nadvazuju.